Test script fails due to path separator // in registry/directory on windows platform

/kind bug

<!--

Welcome! - We kindly ask you to:

  1. Fill out the issue template below 
  2. Use the Google group if you have a question rather than a bug or feature request.

The group is at: https://groups.google.com/forum/#!forum/odo-users

Thanks for understanding, and for contributing to the project!

-->

## What versions of software are you using?

**Operating System:**
Win10

**Output of `odo version`:**
master

## How did you run odo exactly?

Ran `make test-e2e-images` on windows.

## Actual behavior

```
> make test-e2e-images
[...]
SSSSSSSCreated dir: C:\Users\Admin\AppData\Local\Temp\108651876
Creating a new project: asigcqzcof
Running odo.exe with args [odo project create asigcqzcof -w -v4]
[odo]  -  Waiting for project to come up  ...
[odo] I0619 06:50:12.739836    3816 occlient.go:543] Status of creation of project asigcqzcof is Active
[odo] I0619 06:50:12.788612    3816 occlient.go:548] Project asigcqzcof now exists
[odo] I0619 06:50:12.827575    3816 occlient.go:583] Status of creation of service account &ServiceAccount{ObjectMeta:{default  asigcqzcof /api/v1/namespaces/asigcqzcof/serviceaccounts/default 10fa5fca-0585-4a1b-b823-a84cdc72fbf3 103419 0 2020-06-19 06:50:13 +0000 GMT <nil> <nil> map[] map[] [] []  []},Secrets:[]ObjectReference{ObjectReference{Kind:,Namespace:,Name:default-token-6289w,UID:,APIVersion:,ResourceVersion:,FieldPath:,},ObjectReference{Kind:,Namespace:,Name:default-dockercfg-qmhhx,UID:,APIVersion:,ResourceVersion:,FieldPath:,},},ImagePullSecrets:[]LocalObjectReference{LocalObjectReference{Name:default-dockercfg-qmhhx,},},AutomountServiceAccountToken:nil,} is ready
 V  Waiting for project to come up [808ms]
[odo]  V  Project 'asigcqzcof' is ready for use
[odo]  V  New project created and now using project: asigcqzcof
[odo] I0619 06:50:12.852064    3816 odo.go:72] Could not get the latest release information in time. Never mind, exiting gracefully :)



Running oc.exe with args [oc --request-timeout 5m import-image nodejs:latest --namespace=asigcqzcof 
--from=registry.access.redhat.com\rhscl\nodejs-8-rhel7:latest --confirm]

[oc] The ImageStreamImport "nodejs" is invalid: spec.images[0].from.name: Invalid value: 
"registry.access.redhat.com\\rhscl\\nodejs-8-rhel7:latest": invalid reference format



Deleting project: asigcqzcof
Running odo.exe with args [odo project delete asigcqzcof -f]
[odo] I0619 06:50:13.864599   10320 application.go:49] Unable to list Service Catalog instances: unable to list ServiceInstances: the server could not find 
the requested resource (get serviceinstances.servicecatalog.k8s.io)
[odo]  V  Deleted project : asigcqzcof
[odo]  !  Warning! Projects are deleted from the cluster asynchronously. Odo does its best to delete the project. Due to multi-tenant clusters, the project 
may still exist on a different node.
[odo] I0619 06:50:13.989486   10320 odo.go:72] Could not get the latest release information in time. Never mind, exiting gracefully :)
Deleting dir: C:\Users\Admin\AppData\Local\Temp\108651876
------------------------------
+ Failure [2.245 seconds]
odo supported images e2e tests
C:/Users/Admin/go/src/github.com/openshift/odo/tests/e2escenarios/e2e_images_test.go:14
  odo supported images deployment
  C:/Users/Admin/go/src/github.com/openshift/odo/tests/e2escenarios/e2e_images_test.go:83
    Should be able to verify the nodejs-8-rhel7 image [It]
    C:/Users/Admin/go/src/github.com/openshift/odo/tests/e2escenarios/e2e_images_test.go:94
    No future change is possible.  Bailing out early after 0.378s.
    Running oc.exe with args [oc --request-timeout 5m import-image nodejs:latest --namespace=asigcqzcof --from=registry.access.redhat.com\rhscl\nodejs-8-rhel7:latest --confirm]
    Expected
        <int>: 1
    to match exit code:
        <int>: 0
    C:/Users/Admin/go/src/github.com/openshift/odo/tests/helper/helper_run.go:34
------------------------------
[...]
SSS
Summarizing 8 Failures:
[Fail] odo supported images e2e tests odo supported images deployment [It] Should be able to verify the nodejs-8-rhel7 image 
C:/Users/Admin/go/src/github.com/openshift/odo/tests/helper/helper_run.go:34
[...]
Ran 8 of 23 Specs in 20.037 seconds
FAIL! -- 0 Passed | 8 Failed | 0 Pending | 15 Skipped
--- FAIL: TestE2eScenarios (20.52s)
FAIL
```

Failure : The ImageStreamImport "nodejs" is invalid: spec.images[0].from.name: Invalid value: "registry.access.redhat.com\\rhscl\\nodejs-8-rhel7:latest": invalid reference format

## Expected behavior

Image import path should be `registry.access.redhat.com\rhscl\nodejs-8-rhel7:latest`

## Any logs, error output, etc?

--- a/tests/e2escenarios/e2e_images_test.go
+++ b/tests/e2escenarios/e2e_images_test.go
@@ -89,35 +89,35 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the nodejs-10 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", "rhoar-nodejs"+"/"+"nodejs-10:latest", "nodejs:latest", commonVar.Project)
-			verifySupportedImage("rhoar-nodejs"+"/"+"nodejs-10:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "rhoar-nodejs/nodejs-10:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("rhoar-nodejs/nodejs-10:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-10-centos7 image", func() {
-			oc.ImportImageFromRegistry("docker.io", "centos"+"/"+"nodejs-10-centos7:latest", "nodejs:latest", commonVar.Project)
-			verifySupportedImage("centos"+"/"+"nodejs-10-centos7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("docker.io", "centos/nodejs-10-centos7:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("centos/nodejs-10-centos7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12-centos7 image", func() {
-			oc.ImportImageFromRegistry("docker.io", "centos"+"/"+"nodejs-12-centos7:latest", "nodejs:latest", commonVar.Project)
-			verifySupportedImage("centos"+"/"+"nodejs-12-centos7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("docker.io", "centos/nodejs-12-centos7:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("centos/nodejs-12-centos7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 	})
 
 	Context("odo supported images deployment", func() {
 		It("Should be able to verify the openjdk18-openshift image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", "redhat-openjdk-18"+"/"+"openjdk18-openshift:latest", "java:8", commonVar.Project)
-			verifySupportedImage("redhat-openjdk-18"+"/"+"openjdk18-openshift:latest", "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "redhat-openjdk-18/openjdk18-openshift:latest", "java:8", commonVar.Project)
+			verifySupportedImage("redhat-openjdk-18/openjdk18-openshift:latest", "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the openjdk-11-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", "openjdk"+"/"+"openjdk-11-rhel7:latest", "java:8", commonVar.Project)
-			verifySupportedImage("openjdk"+"/"+"openjdk-11-rhel7:latest", "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "openjdk/openjdk-11-rhel7:latest", "java:8", commonVar.Project)
+			verifySupportedImage("openjdk/openjdk-11-rhel7:latest", "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-10-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", "rhscl"+"/"+"nodejs-10-rhel7:latest", "nodejs:latest", commonVar.Project)
-			verifySupportedImage("rhscl"+"/"+"nodejs-10-rhel7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "rhscl/nodejs-10-rhel7:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("rhscl/nodejs-10-rhel7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 	})
 
@@ -130,18 +130,18 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the openjdk-11-rhel8 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", "openjdk"+"/"+"openjdk-11-rhel8:latest", "java:8", "openjdk-11-rhel8")
-			verifySupportedImage("openjdk"+"/"+"openjdk-11-rhel8:latest", "openjdk", "java:8", "openjdk-11-rhel8", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", "openjdk/openjdk-11-rhel8:latest", "java:8", "openjdk-11-rhel8")
+			verifySupportedImage("openjdk/openjdk-11-rhel8:latest", "openjdk", "java:8", "openjdk-11-rhel8", appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", "rhscl"+"/"+"nodejs-12-rhel7:latest", "nodejs:latest", "nodejs-12-rhel7")
-			verifySupportedImage("rhscl"+"/"+"nodejs-12-rhel7:latest", "nodejs", "nodejs:latest", "nodejs-12-rhel7", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", "rhscl/nodejs-12-rhel7:latest", "nodejs:latest", "nodejs-12-rhel7")
+			verifySupportedImage("rhscl/nodejs-12-rhel7:latest", "nodejs", "nodejs:latest", "nodejs-12-rhel7", appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", "ubi8"+"/"+"nodejs-12:latest", "nodejs:latest", "nodejs-12")
-			verifySupportedImage("ubi8"+"/"+"nodejs-12:latest", "nodejs", "nodejs:latest", "nodejs-12", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", "ubi8/nodejs-12:latest", "nodejs:latest", "nodejs-12")
+			verifySupportedImage("ubi8/nodejs-12:latest", "nodejs", "nodejs:latest", "nodejs-12", appName, commonVar.Context)
 		})
 	})
 

--- a/tests/e2escenarios/e2e_images_test.go
+++ b/tests/e2escenarios/e2e_images_test.go
@@ -89,35 +89,35 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the nodejs-10 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("rhoar-nodejs", "nodejs-10:latest"), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.Join("rhoar-nodejs", "nodejs-10:latest"), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("rhoar-nodejs", "nodejs-10:latest")), "nodejs:latest", commonVar.Project)
+			verifySupportedImage(filepath.ToSlash(filepath.Join("rhoar-nodejs", "nodejs-10:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-10-centos7 image", func() {
-			oc.ImportImageFromRegistry("docker.io", filepath.Join("centos", "nodejs-10-centos7:latest"), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.Join("centos", "nodejs-10-centos7:latest"), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("docker.io", filepath.ToSlash(filepath.Join("centos", "nodejs-10-centos7:latest")), "nodejs:latest", commonVar.Project)
+			verifySupportedImage(filepath.ToSlash(filepath.Join("centos", "nodejs-10-centos7:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12-centos7 image", func() {
-			oc.ImportImageFromRegistry("docker.io", filepath.Join("centos", "nodejs-12-centos7:latest"), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.Join("centos", "nodejs-12-centos7:latest"), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("docker.io", filepath.ToSlash(filepath.Join("centos", "nodejs-12-centos7:latest")), "nodejs:latest", commonVar.Project)
+			verifySupportedImage(filepath.ToSlash(filepath.Join("centos", "nodejs-12-centos7:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 	})
 
 	Context("odo supported images deployment", func() {
 		It("Should be able to verify the openjdk18-openshift image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest"), "java:8", commonVar.Project)
-			verifySupportedImage(filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest"), "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest")), "java:8", commonVar.Project)
+			verifySupportedImage(filepath.ToSlash(filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest")), "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the openjdk-11-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("openjdk", "openjdk-11-rhel7:latest"), "java:8", commonVar.Project)
-			verifySupportedImage(filepath.Join("openjdk", "openjdk-11-rhel7:latest"), "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel7:latest")), "java:8", commonVar.Project)
+			verifySupportedImage(filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel7:latest")), "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-10-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("rhscl", "nodejs-10-rhel7:latest"), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.Join("rhscl", "nodejs-10-rhel7:latest"), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("rhscl", "nodejs-10-rhel7:latest")), "nodejs:latest", commonVar.Project)
+			verifySupportedImage(filepath.ToSlash(filepath.Join("rhscl", "nodejs-10-rhel7:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 	})
 
@@ -130,18 +130,18 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the openjdk-11-rhel8 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", filepath.Join("openjdk", "openjdk-11-rhel8:latest"), "java:8", "openjdk-11-rhel8")
-			verifySupportedImage(filepath.Join("openjdk", "openjdk-11-rhel8:latest"), "openjdk", "java:8", "openjdk-11-rhel8", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel8:latest")), "java:8", "openjdk-11-rhel8")
+			verifySupportedImage(filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel8:latest")), "openjdk", "java:8", "openjdk-11-rhel8", appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", filepath.Join("rhscl", "nodejs-12-rhel7:latest"), "nodejs:latest", "nodejs-12-rhel7")
-			verifySupportedImage(filepath.Join("rhscl", "nodejs-12-rhel7:latest"), "nodejs", "nodejs:latest", "nodejs-12-rhel7", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", filepath.ToSlash(filepath.Join("rhscl", "nodejs-12-rhel7:latest")), "nodejs:latest", "nodejs-12-rhel7")
+			verifySupportedImage(filepath.ToSlash(filepath.Join("rhscl", "nodejs-12-rhel7:latest")), "nodejs", "nodejs:latest", "nodejs-12-rhel7", appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", filepath.Join("ubi8", "nodejs-12:latest"), "nodejs:latest", "nodejs-12")
-			verifySupportedImage(filepath.Join("ubi8", "nodejs-12:latest"), "nodejs", "nodejs:latest", "nodejs-12", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", filepath.ToSlash(filepath.Join("ubi8", "nodejs-12:latest")), "nodejs:latest", "nodejs-12")
+			verifySupportedImage(filepath.ToSlash(filepath.Join("ubi8", "nodejs-12:latest")), "nodejs", "nodejs:latest", "nodejs-12", appName, commonVar.Context)
 		})
 	})
 

--- a/tests/e2escenarios/e2e_images_test.go
+++ b/tests/e2escenarios/e2e_images_test.go
@@ -89,35 +89,35 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the nodejs-10 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("rhoar-nodejs", "nodejs-10:latest")), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.ToSlash(filepath.Join("rhoar-nodejs", "nodejs-10:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "rhoar-nodejs"+"/"+"nodejs-10:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("rhoar-nodejs"+"/"+"nodejs-10:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-10-centos7 image", func() {
-			oc.ImportImageFromRegistry("docker.io", filepath.ToSlash(filepath.Join("centos", "nodejs-10-centos7:latest")), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.ToSlash(filepath.Join("centos", "nodejs-10-centos7:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("docker.io", "centos"+"/"+"nodejs-10-centos7:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("centos"+"/"+"nodejs-10-centos7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12-centos7 image", func() {
-			oc.ImportImageFromRegistry("docker.io", filepath.ToSlash(filepath.Join("centos", "nodejs-12-centos7:latest")), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.ToSlash(filepath.Join("centos", "nodejs-12-centos7:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("docker.io", "centos"+"/"+"nodejs-12-centos7:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("centos"+"/"+"nodejs-12-centos7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 	})
 
 	Context("odo supported images deployment", func() {
 		It("Should be able to verify the openjdk18-openshift image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest")), "java:8", commonVar.Project)
-			verifySupportedImage(filepath.ToSlash(filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest")), "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "redhat-openjdk-18"+"/"+"openjdk18-openshift:latest", "java:8", commonVar.Project)
+			verifySupportedImage("redhat-openjdk-18"+"/"+"openjdk18-openshift:latest", "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the openjdk-11-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel7:latest")), "java:8", commonVar.Project)
-			verifySupportedImage(filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel7:latest")), "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "openjdk"+"/"+"openjdk-11-rhel7:latest", "java:8", commonVar.Project)
+			verifySupportedImage("openjdk"+"/"+"openjdk-11-rhel7:latest", "openjdk", "java:8", commonVar.Project, appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-10-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.ToSlash(filepath.Join("rhscl", "nodejs-10-rhel7:latest")), "nodejs:latest", commonVar.Project)
-			verifySupportedImage(filepath.ToSlash(filepath.Join("rhscl", "nodejs-10-rhel7:latest")), "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.access.redhat.com", "rhscl"+"/"+"nodejs-10-rhel7:latest", "nodejs:latest", commonVar.Project)
+			verifySupportedImage("rhscl"+"/"+"nodejs-10-rhel7:latest", "nodejs", "nodejs:latest", commonVar.Project, appName, commonVar.Context)
 		})
 	})
 
@@ -130,18 +130,18 @@ var _ = Describe("odo supported images e2e tests", func() {
 		})
 
 		It("Should be able to verify the openjdk-11-rhel8 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel8:latest")), "java:8", "openjdk-11-rhel8")
-			verifySupportedImage(filepath.ToSlash(filepath.Join("openjdk", "openjdk-11-rhel8:latest")), "openjdk", "java:8", "openjdk-11-rhel8", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", "openjdk"+"/"+"openjdk-11-rhel8:latest", "java:8", "openjdk-11-rhel8")
+			verifySupportedImage("openjdk"+"/"+"openjdk-11-rhel8:latest", "openjdk", "java:8", "openjdk-11-rhel8", appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", filepath.ToSlash(filepath.Join("rhscl", "nodejs-12-rhel7:latest")), "nodejs:latest", "nodejs-12-rhel7")
-			verifySupportedImage(filepath.ToSlash(filepath.Join("rhscl", "nodejs-12-rhel7:latest")), "nodejs", "nodejs:latest", "nodejs-12-rhel7", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", "rhscl"+"/"+"nodejs-12-rhel7:latest", "nodejs:latest", "nodejs-12-rhel7")
+			verifySupportedImage("rhscl"+"/"+"nodejs-12-rhel7:latest", "nodejs", "nodejs:latest", "nodejs-12-rhel7", appName, commonVar.Context)
 		})
 
 		It("Should be able to verify the nodejs-12 image", func() {
-			oc.ImportImageFromRegistry("registry.redhat.io", filepath.ToSlash(filepath.Join("ubi8", "nodejs-12:latest")), "nodejs:latest", "nodejs-12")
-			verifySupportedImage(filepath.ToSlash(filepath.Join("ubi8", "nodejs-12:latest")), "nodejs", "nodejs:latest", "nodejs-12", appName, commonVar.Context)
+			oc.ImportImageFromRegistry("registry.redhat.io", "ubi8"+"/"+"nodejs-12:latest", "nodejs:latest", "nodejs-12")
+			verifySupportedImage("ubi8"+"/"+"nodejs-12:latest", "nodejs", "nodejs:latest", "nodejs-12", appName, commonVar.Context)
 		})
 	})
 

--- a/tests/helper/helper_oc.go
+++ b/tests/helper/helper_oc.go
@@ -3,7 +3,6 @@ package helper
 import (
 	"bufio"
 	"fmt"
-	"path/filepath"
 	"regexp"
 	"strings"
 	"time"
@@ -312,8 +311,8 @@ func (oc OcRunner) checkForImageStream(name string, tag string) bool {
 
 // ImportImageFromRegistry import the required image of the respective component type from the specified registry
 func (oc OcRunner) ImportImageFromRegistry(registry, image, cmpType, project string) {
-	CmdShouldPass(oc.path, "--request-timeout", "5m", "import-image", cmpType, "--namespace="+project, "--from="+registry+"/"+image, "--confirm")
-	CmdShouldPass(oc.path, "annotate", filepath.ToSlash(filepath.Join("istag", cmpType)), "--namespace="+project, "tags=builder", "--overwrite")
+	CmdShouldPass(oc.path, "--request-timeout", "5m", "import-image", cmpType, "--namespace="+project, fmt.Sprintf("--from=%s/%s", registry, image), "--confirm")
+	CmdShouldPass(oc.path, "annotate", fmt.Sprintf("istag/%s", cmpType), "--namespace="+project, "tags=builder", "--overwrite")
 
 }
 

--- a/tests/helper/helper_oc.go
+++ b/tests/helper/helper_oc.go
@@ -312,7 +312,7 @@ func (oc OcRunner) checkForImageStream(name string, tag string) bool {
 
 // ImportImageFromRegistry import the required image of the respective component type from the specified registry
 func (oc OcRunner) ImportImageFromRegistry(registry, image, cmpType, project string) {
-	CmdShouldPass(oc.path, "--request-timeout", "5m", "import-image", cmpType, "--namespace="+project, "--from="+filepath.ToSlash(filepath.Join(registry, image)), "--confirm")
+	CmdShouldPass(oc.path, "--request-timeout", "5m", "import-image", cmpType, "--namespace="+project, "--from="+registry+"/"+image, "--confirm")
 	CmdShouldPass(oc.path, "annotate", filepath.ToSlash(filepath.Join("istag", cmpType)), "--namespace="+project, "tags=builder", "--overwrite")
 
 }

--- a/tests/helper/helper_oc.go
+++ b/tests/helper/helper_oc.go
@@ -312,8 +312,8 @@ func (oc OcRunner) checkForImageStream(name string, tag string) bool {
 
 // ImportImageFromRegistry import the required image of the respective component type from the specified registry
 func (oc OcRunner) ImportImageFromRegistry(registry, image, cmpType, project string) {
-	CmdShouldPass(oc.path, "--request-timeout", "5m", "import-image", cmpType, "--namespace="+project, "--from="+filepath.Join(registry, image), "--confirm")
-	CmdShouldPass(oc.path, "annotate", filepath.Join("istag", cmpType), "--namespace="+project, "tags=builder", "--overwrite")
+	CmdShouldPass(oc.path, "--request-timeout", "5m", "import-image", cmpType, "--namespace="+project, "--from="+filepath.ToSlash(filepath.Join(registry, image)), "--confirm")
+	CmdShouldPass(oc.path, "annotate", filepath.ToSlash(filepath.Join("istag", cmpType)), "--namespace="+project, "tags=builder", "--overwrite")
 
 }
 

--- a/tests/integration/devfile/utils/utils.go
+++ b/tests/integration/devfile/utils/utils.go
@@ -601,7 +601,7 @@ func validateContainerExecListDir(odoV1Watch OdoV1Watch, odoV2Watch OdoV2Watch, 
 			podName := ocRunner.GetRunningPodNameOfComp(odoV1Watch.SrcType+"-app", project)
 			envs := ocRunner.GetEnvs(odoV1Watch.SrcType+"-app", odoV1Watch.AppName, project)
 			dir := envs["ODO_S2I_SRC_BIN_PATH"]
-			stdOut = ocRunner.ExecListDir(podName, project, filepath.Join(dir, "src"))
+			stdOut = ocRunner.ExecListDir(podName, project, filepath.ToSlash(filepath.Join(dir, "src")))
 		}
 	case "docker":
 		dockerRunner := runner.(helper.DockerRunner)

